gbbs/benchmarks/BatchDynamicConnectivity/EulerTourTree/ETTree.h

```diff
@@ -547,8 +547,8 @@ struct ETTree {
     }
 
     template <class KY, class VL, class HH>
-    sequence<sequence<std::pair<uintE, uintE>>> get_subtree_sum(uintE v, uintE parent,
-            pbbslib::sparse_table<KY, VL, HH> edge_index_table) {
+    sequence<sequence<std::pair<uintE, uintE>>> get_subtree_sum(const uintE& v, const uintE& parent,
+            const pbbslib::sparse_table<KY, VL, HH>& edge_index_table) {
```

Comment:
```suggestion
            const pbbslib::sparse_table<KY, VL, HH>& edge_index_table) const {
```
Good! :+1: Consider also making this entire member function `const`: https://stackoverflow.com/a/751690/2769271